original.name="FunctionRef_Invalid_2"
======
>>> main.whiley
function f(int x) -> int:
    return x + 1

function g(function func(int)->byte) -> int:
    return func(1)

method h() -> int
    return g(&f)

---
E300 main.whiley 8,4:9
